File: app/src/main/java/io/github/sithengineer/motoqueiro/ui/statistics/StatisticsContract.java
package io.github.sithengineer.motoqueiro.ui.statistics;

import rx.Observable;

public interface StatisticsContract {
  interface View extends io.github.sithengineer.motoqueiro.View {
    Observable<Void> goHomeClick();
  }

  interface Presenter extends io.github.sithengineer.motoqueiro.Presenter {
    void goHome();
  }
}

File: app/src/main/java/io/github/sithengineer/motoqueiro/ui/statistics/StatisticsContract.kt
package io.github.sithengineer.motoqueiro.ui.statistics

import io.reactivex.Observable

interface StatisticsContract {
  interface View : io.github.sithengineer.motoqueiro.ui.View {
    fun goHomeClick(): Observable<Any>
  }

  interface Presenter : io.github.sithengineer.motoqueiro.ui.Presenter {
    fun goHome()
  }
}
